localized_metadata/metadata_FR.js

```diff
@@ -0,0 +1,76 @@
+/**
```

Comment:
This ain't no France. This is a copy of `metadata_IT.js` to make sure that the `localized_metadata` directory may contain more than one file without complaints from the compiler.